src/identity.ts

```diff
@@ -148,6 +148,64 @@ export const removeTrustedAuthority = async (authorityId: string, options: { ide
     }
 };
 
+export const parseInput = (fileName?: string) => {
+    let data = undefined;
+    if (!fileName) {
+        data = JSON.parse(fs.readFileSync(0, 'utf-8'));
+    }
+    else {
+        if (!fs.existsSync(fileName)) {
```

Comment:
would be cleaner to direcltly use and `else if(!fs.existsSync(fileName)) {` here